vms/platformvm/validators/fee/fee.go

```diff
@@ -91,77 +91,46 @@ func (s State) CostOf(c Config, seconds uint64) uint64 {
 	return cost
 }
 
-// SecondsUntil calculates the number of seconds that it would take to charge at
-// least [targetCost] based on the dynamic fee mechanism. The result is capped
-// at [maxSeconds].
-func (s State) SecondsUntil(c Config, maxSeconds uint64, targetCost uint64) uint64 {
+// SecondsRemaining calculates the maximum number of seconds that a validator
+// can pay fees before their [costLimit] would be exceeded based on the dynamic
```

Comment:
(nit,fyi) The bracket notation in comments has [special treatment](https://tip.golang.org/doc/comment#doclinks) that won't apply here.

Reply:
til